doc/userguide/rules/file-keywords.rst

```diff
@@ -33,24 +37,33 @@ Example::
 
   fileext:"jpg";
 
-filemagic
----------
+file.magic
+----------
 
 Matches on the information libmagic returns about a file.
 
 Syntax::
 
-  filemagic:<string>;
+  file.magic:<string>;
```

Comment:
This incorrect. `filemagic` is the legacy notation, that uses:
`filemagic:"Windows"`;
`file.magic` is the new implementation:
`file.magic; content:"Windows"; nocase;`

Note that like with `filename` above, the difference exists only during parsing in 7.